Components/Utils/ModelRenderer.cs

```diff
@@ -177,8 +179,8 @@ public HtmlString TransactionalFooter()
                 SubTheme = _model.CdtsEnvironment.SubTheme,
                 ShowFooter = false,
                 ContactLinks = _model.Builder.BuildContactLinks(),
-                PrivacyLink = _model.Builder.BuildSingleFooterLink(_model.PrivacyLink),
-                TermsLink = _model.Builder.BuildSingleFooterLink(_model.TermsConditionsLink)
+                PrivacyLink = string.IsNullOrEmpty(_model.PrivacyLink.Href) ? null : new FooterLinkContext { ShowFooter = false, FooterLink = _model.PrivacyLink },
```

Comment:
This is ok, but it side-steps function `Builder.BuildSingleFooterLink` which becomes useless and the logic is duplicated 4 times.  I suggest changing the `Builder.BuildSingleFooterLink` to return `FooterLinkContext` instead (and maybe rename the function to match it new goal).   This function should be used here for both links and in `Footer()`.